client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -306,6 +307,7 @@ export default function NotificationSettings( {
 						selectedDuration={ selectedDuration }
 						selectDuration={ selectDuration }
 					/>
+					<SmsNotification />
```

Comment:
We need to keep this under the feature flag. 

I have added the feature flag for this project - https://github.com/Automattic/wp-calypso/pull/77938.

Also, I'm tempted to keep it as SMSNotification, but I'll leave it up to you.

Reply:
I'm not very familiar with feature flags yet.  But, just verifying nothing needs to be done to this code, it just can't be deployed until the flag is deployed, correct?

Reply:
We need to show the SMS notification only if it is enabled. Now that I have merged the other PR. You can rebase and do something like

```
{ isSMSNotificationEnabled && (
   // Remove the below line and add <SMSNotification />
  <h3 style={ { marginTop: 16 } }>SMS Notification configuration goes here</h3>
) }
```

Reply:
Feature flags are there to prevent it from being shown. If you can see it here - https://github.com/Automattic/wp-calypso/pull/77938. We are allowing it to be shown only in dev env, not others.